runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsPlugin.java

```diff
@@ -155,14 +159,24 @@ void onBoostedLevelChange(BoostedLevelChanged boostedLevelChanged)
 		int last = lastSkillLevels[skillIdx];
 		int cur = client.getBoostedSkillLevel(skill);
 
-		// Check if stat goes +1 or -2
+		// Check if stat goes +1 or -1
 		if (cur == last + 1 || cur == last - 1)
 		{
 			log.debug("Skill {} healed", skill);
 			lastChange = Instant.now();
 			addStatChangeIndicator();
 		}
 		lastSkillLevels[skillIdx] = cur;
+
+		int boostThreshold = config.getBoostThreshold();
```

Comment:
Any reason this is not in the +1/-1 check?

Reply:
I think it's possible for a stat to go down two in the same tick, from normal degen and then snowfall or such.